deployments/Dockerfiles/localnode/app.py

```diff
@@ -122,7 +122,7 @@ def dump_period(self) -> None:
 
 
 def create_app(
-    dump_dir: Optional[Path] = None, perform_monitoring: bool = True
+    dump_dir: Optional[Path] = None, perform_monitoring: bool = True, debug: bool = False
```

Comment:
We're now passing this debug flag so we can set it from the test and generate a lot more info to make the test fail under certain conditions.